app/javascript/components/request-workflow-status/data.js

```diff
@@ -65,12 +65,21 @@ const convertDuration = (duration) => {
 };
 
 const getItemIcon = (item) => {
-  if (item.RunnerContext && item.RunnerContext.success) {
-    return { icon: 'carbon--CheckmarkOutline' };
-  } if (item.RunnerContext && item.RunnerContext.Error) {
-    return { icon: 'carbon--MisuseOutline' };
+  if (item.FinishedTime) {
+    if (item.Output && item.Output.Error) {
+      if (item.RetryCount) {
+        // TODO:
+        return { icon: 'carbon--RetryFailed', tooltip: `${item.Output.Error} : ${item.RetryCount}` }
```

Comment:
tooltip wise, I'm concerned the error will be too big for a tooltip.  If there is no error, will it show ` : 4`?  Also there's no indicator that the trailing number is the retry count.